packages/cli/src/evaluation/test-definitions.service.ee.ts

```diff
@@ -0,0 +1,127 @@
+import { Service } from 'typedi';
+
+import type { TestDefinition } from '@/databases/entities/test-definition.ee';
+import { AnnotationTagRepository } from '@/databases/repositories/annotation-tag.repository.ee';
+import { TestDefinitionRepository } from '@/databases/repositories/test-definition.repository.ee';
+import { BadRequestError } from '@/errors/response-errors/bad-request.error';
+import { NotFoundError } from '@/errors/response-errors/not-found.error';
+import { validateEntity } from '@/generic-helpers';
+import type { ListQuery } from '@/requests';
+
+type TestDefinitionLike = Omit<
+	Partial<TestDefinition>,
+	'workflow' | 'evaluationWorkflow' | 'annotationTag'
+> & {
+	workflow?: { id: string };
+	evaluationWorkflow?: { id: string };
+	annotationTag?: { id: string };
+};
+
+@Service()
+export class TestDefinitionsService {
```

Comment:
This service is `TestDefinitions` while the repository is `TestDefinition` (singular). Could we unify them to use on or the other?

Reply:
Agree and changed both to singular. But I left TestDefinition**s**Controller with plural, so that it reflect API endpoint `/test-definitions`